Source: Rasmus Thomsen < Rasmus.thomsen@protonmail.com >
Reason: We build with buildtype=none, which causes this test to always fail
Upstream: No, distro specific

diff --git a/test/tools/llvm-config/booleans.test b/test/tools/llvm-config/booleans.test
index b28f293..af049a9 100644
--- a/test/tools/llvm-config/booleans.test
+++ b/test/tools/llvm-config/booleans.test
@@ -11,12 +11,6 @@ CHECK-YESNO: {{YES|NO}}
 CHECK-YESNO-NOT: error:
 CHECK-YESNO-NOT: warning
 
-# Also check some other multi-choice options.
-RUN: llvm-config --build-mode 2>&1 | FileCheck --check-prefix=CHECK-BUILD-MODE %s
-CHECK-BUILD-MODE: {{[Dd][Ee][Bb][Uu][Gg]|[Rr][Ee][Ll][Ee][Aa][Ss][Ee]|[Rr][Ee][Ll][Ww][Ii][Tt][Hh][Dd][Ee][Bb][Ii][Nn][Ff][Oo]|[Mm][Ii][Nn][Ss][Ii][Zz][Ee][Rr][Ee][Ll]}}
-CHECK-BUILD-MODE-NOT: error:
-CHECK-BUILD-MODE-NOT: warning
-
 RUN: llvm-config --build-system 2>&1 | FileCheck --check-prefix=CHECK-BUILD-SYSTEM %s
 CHECK-BUILD-SYSTEM: cmake
 CHECK-BUILD-SYSTEM-NOT: error:
